src/types/Dokumentoversikt.ts

```diff
@@ -19,7 +19,13 @@ class Dokumentoversikt {
     }
 
     harGyldigSignatur(): boolean {
-        return this.strukturerteDokumenter.some(({ type }) => type === Dokumenttype.LEGEERKLÆRING);
+        return this.strukturerteDokumenter.some(({ type }) =>
```

Comment:
Påvirker dette pleiepenger på noe vis?

Reply:
nei, de to nye vil bare dukke opp på OLP